### PR TITLE
Put MQTT event under the same category as Event

### PR DESCRIPTION
**Pull request by joostlek**
## Proposed change

SSIA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/event.mqtt.markdown [hunk 1]
```diff
@@ -250,4 +250,4 @@ mqtt:
         {% endfor %} }
 ```
 
-{% endraw %}
\ No newline at end of file
+{% endraw %}
```
Comment (joostlek) [inline source/_integrations/event.mqtt.markdown:253]: Not sure what changed here

Comment (jbouwh) [reply source/_integrations/event.mqtt.markdown:253]: No problem. endraw should be there.

**Review by jbouwh — APPROVED** [review]
LGTM,
Thnx @joostlek 👍

File: source/_integrations/event.mqtt.markdown (hunk 1 above)
Comment (joostlek) [reply source/_integrations/event.mqtt.markdown:253]: Yes, I know, but it was there before as well. Could it be that it's a different line ending?

Comment (jbouwh) [reply source/_integrations/event.mqtt.markdown:253]: Something like that yes. Or a trailing space. Something the lingers missed

**Pull request merged**